src/modules/blocks/ticket/container-header/title/attendee-registration-icons/container.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import Template from './template';
+import { withStore } from '@moderntribe/common/hoc';
+import { selectors } from '@moderntribe/tickets/data/blocks/ticket';
+
+const { select } = wp.data;
+
+const getSaleWindowLabel = ( isTicketFuture, isTicketPast, isTicketOnSale ) => {
+	switch(true) {
+		case isTicketFuture:
+			return 'Scheduled';
+		case isTicketPast:
+			return 'Expired';
+		case isTicketOnSale:
+			return 'On sale';
+	}
```

Comment:
These need to be translated, also use a map, dont pass all 3. Pass a `slug` makes a map easier.

Reply:
Thanks @bordoni, I implemented your suggestion